esp_lcd_touch_xpt2046.c

```diff
@@ -141,6 +160,11 @@ static esp_err_t xpt2046_read_data(esp_lcd_touch_handle_t tp)
     uint32_t x = 0, y = 0;
     uint8_t point_count = 0;
 
+#ifdef CONFIG_XPT2046_INTERRUPT_MODE
+    // Can't check the PENIRQ pin in the driver as the GPIO number isn't available
```

Comment:
I believe it should be available as `tp->config.int_gpio_num`.

Reply:
You are correct. Amendment incoming.